src/views/assets/Asset/AssetCreateUpdate/components/AddAccountDialog.vue

```diff
@@ -78,7 +78,3 @@ export default {
   }
 }
 </script>
-
-<style scoped>
-
-</style>
```

Comment:
I'm sorry, but there is no code provided to analyze. Please attach the code you need help with for proper analysis.